app/validators/passporting_benefit_check/answers_validator.rb

```diff
@@ -10,57 +10,14 @@ def initialize(record)
 
     delegate :errors, :applicant, :crime_application, to: :record
 
-    # Adds the error to the first step name a user would need to go to fix the issue.
-
     def validate
       return true if Passporting::MeansPassporter.new(crime_application).call
```

Comment:
I'm getting an error after i've answered the partner question and it takes me to the task list as the means passporter is checking the applicant is u18 when there is no applicant yet